app/src/androidTest/java/com/hover/stax/MigrationsTest.kt

```diff
@@ -18,7 +18,6 @@ package com.hover.stax
 import androidx.room.testing.MigrationTestHelper
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.platform.app.InstrumentationRegistry
-import com.hover.stax.database.AppDatabase
```

Comment:
This breaks the test - need to import the new import path

Reply:
Which tests cases are broken? Will check this

Reply:
This specific test relies on AppDatabase - so it has a missing import here

```
@get:Rule
    val helper: MigrationTestHelper = MigrationTestHelper(
        InstrumentationRegistry.getInstrumentation(), AppDatabase::class.java
    )
```

Reply:
Alright, will check this